lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -1766,7 +1768,21 @@ def visit_regexp_replace_op_binary(self, binary, operator, **kw):
 class MySQLDDLCompiler(compiler.DDLCompiler):
     def get_column_specification(self, column, **kw):
         """Builds column DDL."""
-
+        if self.dialect.is_mariadb is True:
+            if column.computed is not None and column.nullable is not True:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

we could merge this if with the one above

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4869

Reply:
Hi @CaselIT , 
Thanks for feedback and review,
Updated this in my local while updating realised,
what should be done if someone explicitly specifies `nullable=None`, should exception be raised or it should be defaulted to True in this case, or like raise exception like saying `please specify nullable as False`?.
currently exception will be raised.

Reply:
Looking into documentation, https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.SchemaConst.NULL_UNSPECIFIED

it seems `nullable=None` should not be treated as nullable so did same and updated the testcase you can review.
Added changelog.

Reply:
thanks